app/views/kaui/admin_allowed_users/_form.html.erb

```diff
@@ -3,8 +3,8 @@
     <div class='form-group'>
       <div class="col-sm-offset-2 col-sm-9">
         <div class="checkbox">
-          <%= label_tag :external do %>
-              <%= check_box_tag :external %>Managed externally (LDAP, Okta, etc.)?
+          <%= label_tag :is_managed_externally do %>
+              <%= f.check_box :is_managed_externally, {}, true, false %>Managed externally (LDAP, Okta, etc.)?
```

Comment:
The checkbox should not be editable (when editing, it should be checked or not, but always greyed out).